practice_app/api/tests.py

```diff
@@ -423,6 +424,42 @@ def tearDown(self):
     def test_logout(self):
         self.assertEquals(self.c.get("/api/log-out/").status_code, 200)
 
+class create_paper_list_test_cases(TestCase):
+    def setUp(self): # Setting up a test user object and a test paper list object
+
+        self.c = Client()
+        self.user = User.objects.create_user(
+            username='testuser', password='testpass'
+        )
+
+    def tearDown(self): # When the tests are completed
+        print("POST method tests for creating paper list completed!")
+
+    def test_for_success(self): # Test for successful case
+        h = {"username": "testuser", "password": "testpass", "title": "testlistname1"}
```

Comment:
you should give title in body, not header.

Reply:
fixed